backend/src/main/java/xbdd/webapp/resource/feature/PrintPDF.java

```diff
@@ -1,152 +0,0 @@
-/**
- * Copyright (C) 2015 Orion Health (Orchestral Development Ltd)
- * <p>
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * <p>
- * http://www.apache.org/licenses/LICENSE-2.0
- * <p>
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package xbdd.webapp.resource.feature;
-
-import org.apache.commons.io.IOUtils;
-import xbdd.webapp.util.Coordinates;
-
-import javax.servlet.ServletContext;
-import javax.servlet.http.HttpServletRequest;
-import javax.ws.rs.*;
-import javax.ws.rs.core.Context;
-import javax.ws.rs.core.Response;
-import javax.ws.rs.core.Response.ResponseBuilder;
-import javax.ws.rs.core.StreamingOutput;
-import java.io.*;
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import java.util.Random;
-
-@Path("/print-pdf")
-public class PrintPDF {
```

Comment:
Exporting a document of results is potentially a useful feature of the old product which we currently don't support in the new version. Would it be worth keeping this and instead raising an issue to use it?

Reply:
Yeah can do that, I personally would never print it out but maybe someone would.

Reply:
Yeah we don't really have a need for it, but it might be useful for others. It could be a quick-win if all this backend stuff still works and all we need to do is chuck a button on the frontend somewhere.